atrium/vestibulum/trcflow/flumen/process.go

```diff
@@ -69,37 +66,10 @@ func ProcessFlows(pluginConfig map[string]interface{}, logger *log.Logger) error
 			return err
 		}
 
-		certifyConfig, certifyErr := pluginutil.GetPluginCertifyMap(cGoMod, pluginConfig)
-		if certifyErr != nil {
-			driverConfig.CoreConfig.Log.Println("No certification for plugin:", certifyErr)
-		}
-
-		if newrelic_app_name, ok := certifyConfig["newrelic_app_name"].(string); ok && len(newrelic_app_name) > 0 {
-			if newrelicLicenseKey, ok := certifyConfig["newrelic_license_key"].(string); ok {
-				driverConfig.CoreConfig.Log.Println("Setting up newrelic...")
-				app, err := newrelic.NewApplication(
-					newrelic.ConfigAppName(newrelic_app_name),
-					newrelic.ConfigLicense(newrelicLicenseKey),
-					newrelic.ConfigDistributedTracerEnabled(true),
-					newrelic.ConfigAppLogForwardingEnabled(true),
-					newrelic.ConfigDebugLogger(os.Stdout),
-					newrelic.ConfigInfoLogger(os.Stdout),
-				)
-
-				if err != nil {
-					driverConfig.CoreConfig.Log.Println("Error setting up newrelic:", err)
-					return err
-				}
-
-				driverConfig.CoreConfig.Log = log.New(logWriter.New(driverConfig.CoreConfig.Log.Writer(), app), "["+pluginConfig["pluginName"].(string)+"]", log.LstdFlags)
-				logger = driverConfig.CoreConfig.Log
-				driverConfig.CoreConfig.Log.Println("Newrelic configured...")
-			} else {
-				driverConfig.CoreConfig.Log.Println("Missing license key for newrelic.  Continue without newrelic.")
-			}
-		} else {
-			driverConfig.CoreConfig.Log.Println("Missing app name for newrelic.  Continue without newrelic.")
-		}
+		// TODO: should these have capabilities?
+		pluginConfig["pluginName"] = pluginNameList[0]
```

Comment:
ok check?

Reply:
We'll fix this post merge and make another pr.